src/pl/plpython/plpy_main.c

```diff
@@ -80,6 +82,24 @@ cancel_pending_hook_type prev_cancel_pending_hook;
 void PLy_handle_cancel_interrupt(void);
 
 bool PLy_enter_python_intepreter = false;
+bool PLy_path_added = false;
+
+/* GUC variables */
+#if PY_MAJOR_VERSION >= 3
+char *plpython3_path = NULL;
```

Comment:
make it static, it should not be used out of this file.

```
static char *plpython3_path = NULL;
```

Reply:
copy that.